src/main/resources/application.yaml

```diff
@@ -88,6 +88,7 @@ featureFlag:
   isfeePaymentEnabled: ${IA_FEEPAYMENT_ENABLED:false}
   isSaveAndContinueEnabled: ${IS_SAVE_AND_CONTINUE_ENABLED:true}
   isHomeOfficeIntegrationEnabled: ${IA_HOME_OFFICE_INTEGRATION_ENABLED:false}
+  isReheardAppealEnabled: ${IA_REHEARD_APPEAL_ENABLED:false}
```

Comment:
can we use Launch Darkly flag system for it? F&P and HO go old way but new epics should go with Launch Darkly.

Reply:
This is now in place.